site/hxapi/transfers/view.md

```diff
@@ -0,0 +1,290 @@
+| Destination          | Destination point of the transfer.                                            |
+| DestinationAddress   | Address of the destination location.                                          |
+| DestinationIata      | IATA code of the destination location.                                        |
+| JourneyTime          | Duration of the journey in minutes.                                           |
+| FlightDetails        | Flight information associated with the transfer.                              |
+| Pickup               | Pickup time for the transfer vehicle as an ISO 8601 timestamp in UTC (Zulu time). |
+| InformationUrl       | URL with additional transfer information (when available from supplier).     |
+| JoiningInstruction   | Instructions on where and how to join the transfer.                          |
+| ContactNumbers       | Contact information including UkEmergencyTel, ReconfirmationTel, and EmergencyTel phone numbers. |
+
+### FlightDetails Fields
+
+The `FlightDetails` object contains flight information when the transfer is associated with a flight:
+
+| Field                                | Additional Information                                                        |
+|--------------------------------------|-------------------------------------------------------------------------------|
+| FlightDetails/Code                   | Flight number/code (e.g., "U28605").                                         |
+| FlightDetails/Origin                 | Origin airport IATA code (e.g., "LGW").                                      |
+| FlightDetails/Destination            | Destination airport IATA code (e.g., "AGP").                                 |
+| FlightDetails/Departure              | Flight departure time as ISO 8601 timestamp in UTC (Zulu time).              |
+| FlightDetails/Arrival                | Flight arrival time as ISO 8601 timestamp in UTC (Zulu time).                |
+
+**Important Notes:**
+- All timestamps use UTC timezone and are formatted as ISO 8601 with 'Z' suffix (e.g., "2025-08-01T10:00:00Z")
+- For airport pickup transfers, the `Arrival` time represents when passengers land and need pickup
+- For airport drop-off transfers, the `Departure` time represents when passengers need to be at the airport
+- The `FlightDetails` object and its individual fields can be null when flight information is not available or unknown
+
+### Example
+
+{% codetabs %}
+{% codetab XML %}
+```xml
+<Booking>
+  <BookingRef>TBBGNHMT</BookingRef>
+  <SupplierRef>123456</SupplierRef>
+  <Status>Confirmed</Status>
+  <DateBooked>2025-01-15T10:30:00Z</DateBooked>
+  <DateCancelled/>
+  <Email>customer@example.com</Email>
+  <Product>
+    <Price>103.52</Price>
+    <Currency>GBP</Currency>
+    <Category>private_car</Category>
+    <TransferCode>TBHT1169922</TransferCode>
+    <DeparturePointType>Airport</DeparturePointType>
+    <ArrivalPointType>Hotel</ArrivalPointType>
```

Comment:
These fields are slightly odd, because in fact they apply to the OutboundTransfer, but they're reversed for the ReturnTransfer.
Maybe they're even redundant, since we include IATAs or addresses in these two transfer blocks?